deploy-manage/users-roles/custom-roles.md

```diff
@@ -1,7 +1,7 @@
 ---
 mapped_urls:
   - https://www.elastic.co/guide/en/serverless/current/custom-roles.html
-applies:
+applies_to:
```

Comment:
Maybe changing the navigation title to `Serverless projects custom roles` to ensure it doesn't cause confusion?